gravitee-apim-cypress/cypress/integration/rest-api/apis/mock-policy-flow.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER } from 'fixtures/fakers/users/users';
+import { ApiFakers } from 'fixtures/fakers/apis';
+import { PlanFakers } from 'fixtures/fakers/plans';
+import { createApi, deleteApi, deployApi, startApi, stopApi, updateApi } from 'commands/management/api-management-commands';
+import { createPlan, publishPlan, deletePlan } from 'commands/management/api-plan-management-commands';
+import { Api } from 'model/apis';
+import { NewPlanEntity, SecurityType } from 'model/plan';
+
+context('Create a mock policy', () => {
+  let createdApi: Api;
+  let createdPlan: NewPlanEntity;
+
+  before(() => {
+    const fakeApi: Api = ApiFakers.api();
+    const fakePlan = PlanFakers.plan({ security: SecurityType.KEY_LESS });
+    createApi(ADMIN_USER, fakeApi)
+      .created()
+      .then((response) => {
+        createdApi = response.body;
+        createPlan(ADMIN_USER, createdApi.id, fakePlan)
+          .created()
+          .its('body')
+          .then((plan: NewPlanEntity) => {
+            createdPlan = plan;
+            publishPlan(ADMIN_USER, createdApi.id, createdPlan.id).ok();
+          });
+      });
+  });
+
+  it('should create and deploy a mock policy', () => {
+    cy.fixture('json/updateApi_mockPolicy_example').then((apiFixture) => {
+      let mockPolicyApi = {};
+      mockPolicyApi.name = createdApi.name;
+      mockPolicyApi.description = createdApi.description;
+      mockPolicyApi.proxy = createdApi.proxy;
+      mockPolicyApi.version = createdApi.version;
+      mockPolicyApi.visibility = createdApi.visibility;
+      mockPolicyApi.path_mappings = apiFixture.path_mappings;
+      mockPolicyApi.paths = apiFixture.paths;
+      updateApi(ADMIN_USER, createdApi.id, mockPolicyApi).ok();
+      deployApi(ADMIN_USER, createdApi.id).ok();
+      startApi(ADMIN_USER, createdApi.id).noContent();
+    });
+  });
+
+  it('should successfully call the mocked API endpoint', { retries: 50 }, function () {
```

Comment:
`retries` is only available when invoked via the mocha CLI, so I don't know how it's done internally in cypress

more over, in my opinion, we have to find a better way to do that. `retries` should not be at `it()` level but at request's one directly.

Reply:
Ok, thanks for the hint.
By now I found a different solution for that. The next update of the PR will have a custom Cypress command to deal with request that needs retries.